builtin/providers/google/resource_storage_object_acl.go

```diff
@@ -150,18 +150,9 @@ func resourceStorageObjectAclRead(d *schema.ResourceData, meta interface{}) erro
 		}
 
 		for _, v := range res.Items {
-			role := ""
-			entity := ""
-			for key, val := range v.(map[string]interface{}) {
-				if key == "role" {
-					role = val.(string)
-				} else if key == "entity" {
-					entity = val.(string)
-				}
-			}
-			if _, in := re_local_map[entity]; in {
-				role_entity = append(role_entity, fmt.Sprintf("%s:%s", role, entity))
-				log.Printf("[DEBUG]: saving re %s-%s", role, entity)
+			if _, in := re_local_map[v.Entity]; in {
+				role_entity = append(role_entity, fmt.Sprintf("%s:%s", v.Role, v.Entity))
+				log.Printf("[DEBUG]: saving re %s-%s", v.Role, v.Entity)
```

Comment:
I'm not sure changing this file is necessary anymore- we've definitely revendored since November